Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:19-alpine
```

Comment:
ensure the same jdk version is used across the project

Reply:
confirmed we're sticking to 17